core/src/main/java/com/radixdlt/mempool/MempoolRelayer.java

```diff
@@ -132,7 +129,7 @@ public EventProcessor<MempoolRelayTrigger> mempoolRelayTriggerEventProcessor() {
     return ev -> {
       final var transactions =
           this.mempoolRelayReader.getTransactionsToRelay(
-              MAX_RELAY_MSG_NUM_TXNS, MAX_RELAY_MSG_TOTAL_TXN_PAYLOAD_SIZE);
+              (int) this.maxMessageTransactionCount, (int) this.maxMessagePayloadSize);
```

Comment:
`Ints.checkedCast()` would make my paranoid side feel safer :p

alternatively (maybe even better), could it be parsed as `int` to begin with?